docs/src/css/colors.css

```diff
@@ -98,7 +96,7 @@
   --nav-column-link: var(--neutral-dark-2);
   --nav-column-link-hover: var(--neutral-dark-3);
 
-  --scrubber-hover: var(--primary-gradient-alt-1);
+  --scrubber-hover: var(--primary-gradient);
```

Comment:
bit confused, looks like `--primary-gradient` was deleted? so what is this referencing

Reply:
ah unused. thx for catching!! this is in docs